django/contrib/admin/static/admin/css/rtl.css

```diff
@@ -199,6 +199,16 @@ fieldset .fieldBox {
     text-align: center;
 }
 
+.calendar .yearlist a {
+	padding-left: 0px;
+	padding-right: 15px;
+}
+
+.calendar .yearlist a.toyear {
+	padding-left: 0px;
+	padding-right: 15px;
+}
+
```

Comment:
I would recommend writing those styles with `padding-inline-start: 15px;` and `padding-inline-end: 15px;` directly in the main stylesheet so we don’t have to keep those two sets of styles in sync. I believe this will be compatible with Django’s stated browser support.

Reply:
tnx, padding-inline-start replaced instead padding-left and padding-right.